app/src/main/res/values/colors.xml

```diff
@@ -3,4 +3,7 @@
     <color name="colorPrimary">#3F51B5</color>
     <color name="colorPrimaryDark">#303F9F</color>
     <color name="colorAccent">#FF4081</color>
+
+
+    <color name="secureBtn">#5c5c5c</color>
```

Comment:
This might benefit from a more descriptive name because we're not sure if `secureBtn` refers to the background color or the text color. Maybe `secureButtonBackgroundColor`?